nrf-hal-common/src/monotonic.rs

```diff
@@ -1,13 +1,46 @@
 /*!
 Implements the [Monotonic](rtic_monotonic::Monotonic) trait for the TIMERs and the RTCs.
 
-<Här borde vi ha användings exempel>
+<Strong> Example (RTC): </Strong>
+```rust
```

Comment:
Change to `ignore` instead of `rust` since this code is not a valid rust program in and of itself this would not pass `cargo test`. and should therefore not be run in a cargo test. So replacing `rust` with `ignore` will enforce that this is not to be run during testing. 
https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html

Reply:
fixed